packages/library/build.gradle.kts

```diff
@@ -217,7 +217,7 @@ realmPublish {
 }
 
 tasks.dokkaHtml.configure {
-    moduleName.set("Realm Kotlin Multiplatform SDK")
+    moduleName.set("realm-kotlin")
```

Comment:
Isn't this used in the title? https://docs.mongodb.com/realm-sdks/kotlin/latest/

Reply:
Not 100% sure why this needs to change? But if it does we also need to warn @nathan-contino-mongo as he currently depends on it for some links.

Reply:
Yeah. It was just because the name is also used in the URL and for breadcrumbs so I found it better with a short name. I don't think there is a way to have a different module name and title, so will revert to avoid having to update the docs link.